e2e/cp_test.go

```diff
@@ -4196,3 +4197,42 @@ func TestCountingWriter(t *testing.T) {
 	expected := fs.Expected(t, fs.WithFile(filename, content, fs.WithMode(0644)))
 	assert.Assert(t, fs.Equal(cmd.Dir, expected))
 }
+
+// It should skip special files
+func TestUploadingSocketFile(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	workdir := fs.NewDir(t, t.Name())
+	defer workdir.Remove()
+	sockaddr := workdir.Path() + "/s5cmd.sock"
+	ln, err := net.Listen("unix", sockaddr)
+	if err != nil {
+		t.Log(err)
+	}
```

Comment:
Test should be aborted if socket creation fails, nothing to copy if file does not exist.

Reply:
Resolved. https://github.com/peak/s5cmd/pull/618/commits/cd565309bf96dbb53efa2f356cfa974280fbf287